### PR TITLE
fix: add Muhammad Sameem

### PR DESCRIPTION
**Pull request by sameem420**
## Changes proposed

- ### add sameem420

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/879"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add sameem420`

File: public/data/sameem420.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Muhammad Sameem",
+  "type": "personal",
+  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech & an open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects ",
```
Comment (Panquesito7) [inline public/data/sameem420.json:4]:
```suggestion
  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

**Review by syedsohan — APPROVED** [review]

File: public/data/sameem420.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Muhammad Sameem",
+  "type": "personal",
+  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```
Comment (krishguptadev) [inline public/data/sameem420.json:4]: IMHO Having all caps in b/w of the text seems weird.

```suggestion
  "bio": "A JavaScript Dev (MERN Stack + React Native) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

File: public/data/sameem420.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "icon": "dollar",
+      "color": "red",
+      "description": "Started working as a React Developer for an US-based StartUP"
+    },
+    {
+      "title": "Started some OS Projects",
+      "date": "December 2021",
+      "icon": "globe",
+      "color": "blue",
+      "description": "Building some projects for community use. Currently in progress but soon will launch them and they will be open for contributions"
```
Comment (krishguptadev) [inline public/data/sameem420.json:43]:
```suggestion
      "description": "Building some projects for community use. Currently in progress but soon will launch them and they will be open for contributions!"
```

File: public/data/sameem420.json (hunk 2 above)
Comment (syedsohan) [inline public/data/sameem420.json:4]:
```suggestion
  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/sameem420.json (hunk 2 above)
Comment (krishguptadev) [reply public/data/sameem420.json:4]: I think the previous one is totally fine because the bio is really long already 😅

Comment (syedsohan) [reply public/data/sameem420.json:4]: There's double `and` (& and)! That's why I asked to remove one AND 😄

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**